### ISSUE TITLE
We might want to update the default probability threshold to be higher than 0

### ISSUE THREAD

**Issue by 36000** [yeatmanlab/pyAFQ#984]
https://github.com/yeatmanlab/pyAFQ/blob/f70b18d54bcd0a6086db85c2722fc8f135225b7a/AFQ/segmentation.py#L131

**Comment by arokem** [comment]
Can you say more? What would be a more appropriate threshold default? 

The logic behind this was that the probability maps serve as a very lenient initial filter, so a streamline that passes through any voxel that has non-zero probability of belonging to a particular bundle goes to the next stage of consideration. This is actually a very effective filter, because for many bundles it gets rid of a very large number of streamlines.

**Comment by 36000** [comment]
Yes, but I think in practice a threshold of, say, 0.1 (10%) will exclude a lot of aberrant streamlines that just barely touch the map, and not exclude any streamlines we want to keep. I noticed this making the figures for the review paper, that with our current threshold the probability maps are extremely lenient.

Maybe this is OK, they still do thin out the tractography considerably, as you point out.
